SmartCar.ino

```diff
@@ -1,5 +1,20 @@
 #include <Smartcar.h>
 
+
+const int yellowRight = 20;// declaring LEDpins.
+const int redRight = 18;
+const int yellowLeft = 21;
+const int redLeft = 19;
+
+unsigned long yellowCurrent,redCurrent,yellowTurningCurrent,redTurningCurrent,redBackingCurrent,yellowBackingCurrent; // Declaring the different Current millis.
+unsigned long previousYellowDrive,yellowTurnPrev,redTurnPrev, previousRedDrive,yellowBackPrev,redBackPrev = 0; // Declaring  the different Previous Millis.

```

Comment:
Declaring variables like this (in the same line, separated by commas) is generally considered a bad practice. You might think that the `= 0` in the end initializes their value to `0` but it does not necessarily do that.